pkg/reconcilermanager/controllers/reconciler_base.go

```diff
@@ -501,6 +503,22 @@ func mutateContainerResource(c *corev1.Container, override *v1beta1.OverrideSpec
 			}
 		}
 	}
+	var found = false
+	for _, logLevel := range override.LogLevel {
+		if logLevel.ContainerName == c.Name {
+			c.Args = append(c.Args, fmt.Sprintf("--v=%d", logLevel.LogLevel))
+			found = true
+		}
+	}
+
+	if !found {
+		if c.Name == "git-sync" {
+			// git-sync container's default is 5
+			c.Args = append(c.Args, fmt.Sprintf("--v=%d", 5))
```

Comment:
Might be nice if these defaults were pulled out somewhere so they could be managed in one place, like maybe a `containerLogLevelDefault(containerName string)` function. That would slightly simplify the logic here.

Reply:
done. created and used `containerLogLevelDefault(containerName string)` function